scripts/create-external-bootstrap-secret.sh

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -o errexit
+
+
+TMPFOLDER="$(mktemp -d -p . "kubestellar-XXXX")"
+trap 'rm -rf "$TMPFOLDER"' EXIT
+BOOTSTRAP_KUBECONFIG="$TMPFOLDER/bootstrap-config"
+
+
+# Colors
+COLOR_NONE="\033[0m"
+COLOR_RED="\033[1;31m"
+COLOR_GREEN="\033[1;32m"
+COLOR_BLUE="\033[94m"
+COLOR_YELLOW="\033[1;33m"
+COLOR_PURPLE="\033[1;35m"
+COLOR_WARNING="${COLOR_RED}"
+COLOR_ERROR="${COLOR_RED}"
+COLOR_STATUS_TRUE="${COLOR_GREEN}"
+COLOR_STATUS_FALSE="${COLOR_RED}"
+COLOR_INFO="${COLOR_BLUE}"
+COLOR_TITLE="${COLOR_BLUE}"
+COLOR_YAML="${COLOR_PURPLE}"
+
+
+# Command line arguments
+arg_cp=""
+KUBECONFIG="${KUBECONFIG:-$HOME/.kube/config}"
+arg_context=""
+arg_source_kubeconfig=""
+arg_source_context=""
+arg_ns="default"
+arg_addr=""
+arg_verbose=false
+
+
+# Display help
+display_help() {
+  cat << EOF
+Usage: $0 [options]
+
+--controlplane|-c <name>       control plane name used to name the secret: <name>-bootstrap
+--namespace|-n <name>          namespace name where to create the secret, default is "default"
+--kubeconfig|-K <filename>     use the specified kubeconfig used to generate the bootstrat secret
```

Comment:
"generate" is a bit odd here, it calls to mind computation/processing. I think "store" is more to the point.